### PR TITLE
Fix issue #87

### PR DESCRIPTION
**Pull request by cjh79**
Only configure hammer swipe options for direction specified in ba-swipe-close attribute. Fixes issue #87 with touch scrolling failing when using ba-swipe-close=left or right.

### CODE REVIEW

File: js/angular/components/common/common.js [hunk 1]
```diff
@@ -129,32 +129,39 @@
...
+      if (typeof(Hammer)!=='undefined') {
+        hammerElem = new Hammer(element[0]);
+        // set the options for swipe (to make them a bit more forgiving in detection)
+        hammerElem.get('swipe').set({
+          direction: hammerDirection,
+          threshold: 5, // this is how far the swipe has to travel
+          velocity: 0.5 // and this is how fast the swipe must travel
+        });
       }
       if(typeof(hammerElem) !== 'undefined'){
```
Comment (soumak77) [inline js/angular/components/common/common.js]: This if statement can be removed and the body can be moved inside the if on line 157

**Review by soumak77 — APPROVED** [review]
Requested change refers to old code, so not mandatory that it gets addressed

**Comment by cjh79** [comment]
v2.0.8 says it contains this fix, but I don't see it?

https://github.com/base-apps/angular-base-apps/releases/tag/v2.0.8

**Comment by soumak77** [comment]
@cjh79 here's the merge: https://github.com/base-apps/angular-base-apps/commit/3dbaebee749177ce115ed3cc202cb457b2713b2f

**Comment by cjh79** [comment]
I see the fix in the repo, but it is not in the release for some reason. The tar.gz [here](https://github.com/base-apps/angular-base-apps/releases/tag/v2.0.8) does not contain the fix, nor does the 2.0.8 version I retrieved via bower. Am I missing something?

**Comment by cjh79** [comment]
Hi, any idea what's going on here? That code is not in the release.

**Pull request merged**